l10n/de/toolkit/chrome/mozapps/extensions/extensions.dtd

```diff
@@ -22,7 +22,8 @@
 <!ENTITY installAddonFromFile.accesskey       "A">
 <!ENTITY toolsMenu.tooltip                    "Tools für alle Add-ons">
 <!ENTITY recommendedAddon.label               "Recommended Add-ons">
-<!ENTITY installedAddon.label                 "Installed Add-ons">
+<!ENTITY installAddonDisclaimer.label         "Vorsicht beim Installieren von Add-ons. ">
```

Comment:
Do we really need this extra blank char after "Add-ons." ?

Reply:
Nope.